docs/voyage_image_building_alix.md

```diff
@@ -0,0 +1,157 @@
+Building Voyage Linux image for PC Engines ALIX platforms
+=========================================================
+
+Intro
+-----
+* Example DIRECTORY_TO_CF_CARD_DEVICE used in instruction: `/dev/sdc`
+* Voyage Linux image downloaded from `http://pcengines.ch` website has no
+`flashrom` and `cbmem` installed
+* Voyage Linux kernel of image downloaded from `http://pcengines.ch` website has
+`CONFIG_CPU_FREQ` option not set, which could be required for `coreboot 4.7.x`
+* Linux Voyage default login/password: `root/voyage`
+* Linux Voyage default serial console baudrate: `38400`
+
+Requirements
+------------
+* Compact Flash with a minimum of 1 GB of memory
+* Compact Flash cards reader
+
+Flashing CF card with Voyage Linux image from `http://pcengines.ch`
+-------------------------------------------------------------------
+
+1. Download [Voyage Linux image for Alix platforms](http://pcengines.ch/file/voyage-0.9.2.img.zip).
+
+2. Unmount all mounted in OS partitions of used CF card:
+
+```
+sudo umount <DIRECTORY_TO_CF_CARD_DEVICE>*
+```
+Eg.:
+```
+sudo umount /dev/sdc*
+```
+
+3. Unzip downloaded file and Flash your CF flash with unzipped image:
+
+```
+unzip -p voyage-0.9.2.img | pv | sudo dd of=<DIRECTORY_TO_CF_CARD_DEVICE> bs=16M
+
+```
+
+Eg.:
+```
+unzip -p voyage-0.9.2.img | pv | sudo dd of=/dev/sdc bs=16M
+```
+
+After succesful CF card flashing information similar to shown below may appear:
+```
+60+1 records in
+60+1 records out
+1014644736 bytes (1.0 GB, 968 MiB) copied, 32.4125 s, 31.3 MB/s
+```
```

Comment:
@acihy just removing card at this point may cause problem. This is known to all storage media USB, SD etc. You should run `sync` to drop cache to device. `watch grep -e Dirty: -e Writeback: /proc/meminfo` command is very useful to check if cache was cleaned. I spent countless hours debugging problems with not correctly dumped system image.